### ISSUE TITLE
Missing podman plugin in sosreport

### ISSUE THREAD

**Issue by EmilienM** [containers/libpod#1365]
> /kind feature

**Description**

sosreport doesn't have a plugin for podman, like we have for [docker](https://github.com/sosreport/sos/blob/db7a86e64c7b6e35f3752974381b52be05dd712d/sos/plugins/docker.py).
It would be essential to have the plugin to support and debug podman in the future.

**Comment by rhatdan** [comment]
Yes @EmilienM Please open this in https://github.com/sosreport/sos

**Comment by EmilienM** [comment]
@vrothberg it's perfectly fine!

**Comment by EmilienM** [comment]
FTR https://github.com/sosreport/sos/pull/1408
